tests/sentry/tasks/test_commit_context.py

```diff
@@ -408,3 +411,157 @@ def after_return(self, status, retval, task_id, args, kwargs, einfo):
                 )
 
             assert mock_suspect_commits.called
+
+
+@region_silo_test(stable=True)
+@patch(
+    "sentry.integrations.github.GitHubIntegration.get_commit_context",
+    Mock(
+        return_value={
+            "commitId": "asdfwreqr",
+            "committedDate": "2023-02-14T11:11Z",
```

Comment:
Isn't it better to add a date() here so you make sure in tests the commit is always recent?

Reply:
the commit date isn't important in this case. I'd rather just mock a response once here and have the object be fixed.